src/modules/notifications/notifications.service.ts

```diff
@@ -35,6 +37,29 @@ export class NotificationsService {
     private userRepository: Repository<User>
   ) {}
 
+  async createGlobalNotifications(dto: CreateNotificationForAllUsersDto) {
+    const users = await this.userRepository.find();
+    try {
+      const notifications = users.map(user => {
+        return this.notificationRepository.create({
+          message: dto.message,
+          user,
+        });
+      });
+      await this.notificationRepository.save(notifications);
+      return {
+        status: 'success',
+        message: 'Notification created successfully',
+        data: null,
+      };
+    } catch (error) {
```

Comment:
you need not handle server errors within the modules. we have an interceptor in place for this. You can remove the try / catch confidently

Reply:
Noted. Removed the try catch